### PR TITLE
fix exception while loading images: handle not found resp. content

### PR DESCRIPTION
**Pull request by resihoehne**
I was facing some issues when following the current readme. 
I managed to download sheets after updating the suggested browser console commands
( btw: For the song I used for testing, the image link did not change to the one shown in the sources ) 
and  caught the response content inside the while loop which is returned after the last image. 
In my test case "No content" was returned.
With this, I was able to run the app without any exception again :)
I also recommend to suggest using virtual environments when using a local version of the repo.

### CODE REVIEW

**Comment by hannah-jj** [comment]
@resihoehne  Do you mean "Not Found"?

**Comment by resihoehne** [comment]
Indeed, "Not found" is returned. Just a typo in my comment. I correctly added that to the while loop :)

File: README.md
Comment (MatthiasLienhard) [inline README.md:1]: This change does not do anything, and is harder to analyse if something goes wrong.

**Review by MatthiasLienhard — APPROVED** [review]
I'll merge the changes except for the readme

**Pull request merged**